var/spack/repos/builtin/packages/py-pylint/package.py

```diff
@@ -30,8 +30,10 @@ class PyPylint(PythonPackage):
     homepage = "https://pypi.python.org/pypi/pylint"
     url      = "https://pypi.python.org/packages/source/p/pylint/pylint-1.4.1.tar.gz"
```

Comment:
Replace this URL with:
https://pypi.io/packages/source/p/pylint/pylint-1.7.2.tar.gz
and you won't need to add a version-specific URL below.